Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -64,7 +64,10 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
 
             if (user.get('avatar')) {
                 contentService.loadImageVariation(user.get('avatar').variations.platformui_profileview.href, tasks.add(function (error, response) { //jshint ignore:line
-                    if ( error ) {
+                    if ( error && error.message.indexOf('401') !== -1 ) {
+                        error = false;
```

Comment:
I don't see the point in modifying `error` which is a callback parameter. I think this can be simplified in something like:

```js
if ( error && response.status !== 401 ) {
    // 401 error is ignored because the user might not have the right to read
    // his/her own avatar see https://jira.ez.no/browse/EZP-25522
    loadError = true;
    return;
}
```